tests/unit/browser/network/test_pastebin.py

```diff
@@ -0,0 +1,119 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2016 Anna Kobak (avk) <awerk@onet.eu>:
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for qutebrowser.browser.network."""
+
+import pytest
+from PyQt5.QtCore import QUrl
+
+from qutebrowser.browser.network import pastebin
+from qutebrowser.misc import httpclient
+
+
+class HTTPPostStub(httpclient.HTTPClient):
```

Comment:
Is there a reason you inherit from `httpclient.HTTPClient` here? I think it's not needed, and it's error prone (accidental network requests when forgetting to override something).